docs/.benchmarks_table.md

```diff
@@ -2,10 +2,11 @@
 
 Package | Version | Relative Performance | Mean validation time
 --- | --- | --- | ---
-pydantic | `1.1` |  | 43.6μs
-attrs + cattrs | `19.3.0` | 1.4x slower | 59.9μs
-valideer | `0.4.2` | 1.4x slower | 61.8μs
-marshmallow | `3.2.2` | 2.5x slower | 107.6μs
-trafaret | `2.0.0` | 3.4x slower | 148.7μs
-django-rest-framework | `3.10.3` | 12.6x slower | 551.2μs
-cerberus | `1.3.2` | 26.3x slower | 1146.3μs
+attrs + cattrs | `19.3.0` |  | 16.1μs
+valideer | `0.4.2` | 1.3x slower | 21.1μs
+pydantic | `1.5.1` | 4.3x slower | 69.8μs
```

Comment:
are you using the compiled version of pydantic? Does you machine match that's in the docs?

Reply:
Apparantly I was not using a compiled version.

Running the following outputs `False` on my machine:
```
python3.7 -m venv .venv
source .venv/bin/activate
pip install pydantic
python -c 'import pydantic; print(pydantic.compiled)'                                                                                                                                                                                          
```
I'm running Ubuntu 18.04.4 LTS and Python 3.7.5.

When compiling with cython locally, pydantic goes from 4.3x to 2.7x slower.